docs/core/tools/dotnet-install-script.md

```diff
@@ -22,34 +22,31 @@ ms.assetid: b64e7e6f-ffb4-4fc8-b43b-5731c89479c2
 
 Windows:
 
-`dotnet-install.ps1 [-Channel] [-Version] [-InstallDir] [-Architecture] [-SharedRuntime] [-DebugSymbols] [-DryRun] [-NoPath] [-AzureFeed] [-ProxyAddress]`
+`dotnet-install.ps1 [-Channel] [-Version] [-InstallDir] [-Architecture] [-SharedRuntime] [-DryRun] [-NoPath] [-AzureFeed] [-ProxyAddress] [--Verbose] [--Help]`
 
 macOS/Linux:
 
-`dotnet-install.sh [--channel] [--version] [--install-dir] [--architecture] [--shared-runtime] [--debug-symbols] [--dry-run] [--no-path] [--verbose] [--azure-feed] [--help]`
+`dotnet-install.sh [--channel] [--version] [--install-dir] [--architecture] [--shared-runtime] [--dry-run] [--no-path] [--azure-feed] [--verbose] [--help]`
 
 ## Description
 
-The `dotnet-install` scripts are used to perform a non-admin install of the CLI toolchain and the shared runtime. You can download the scripts from the [CLI GitHub repo](https://github.com/dotnet/cli/tree/rel/1.0.0/scripts/obtain). 
+The `dotnet-install` scripts are used to perform a non-admin install of the CLI toolchain and the shared runtime.
```

Comment:
I know you haven't changed this part but some customers have been complaining we use different terminology for the same things. Doesn't CLI toolchain + shared runtime represent the .NET Core SDK? If so, I'd change to that.

If that's the case, you could say something like "non-admin installation of the .NET Core SDK, which includes the .NET Core CLI tools and the shared runtime."

Reply:
done...